neuralplexer/inference.py

```diff
@@ -223,12 +223,20 @@ def multi_pose_sampling(
                     plddt_lig_all.append(None)
                 else:
                     plddt_lig_all.append(plddt_lig[struct_idx].item())
+    if confidence and args.rank_outputs_by_confidence:
```

Comment:
@zrqiao, per your comment about making ligand lDDT ranking (e.g., for docking) a CLI option, the `args.rank_outputs_by_confidence` flag now allows users to decide whether or not they would like the predicted complexes for each complex input to be ranked.